tests/test_output_formatter.cpp

```diff
@@ -1373,12 +1597,266 @@ const std::vector<FormatterParamType> non_orderable_list_info_formatter_outputs{
      "            \"mounts\": {\n"
      "            },\n"
      "            \"release\": \"\",\n"
-     "            \"snapshots\": \"3\",\n"
+     "            \"snapshot_count\": \"3\",\n"
      "            \"state\": \"Stopped\"\n"
      "        }\n"
      "    }\n"
      "}\n",
      "json_info_multiple_instances"},
+    {&json_formatter, &single_snapshot_info_reply,
+     "{\n"
+     "    \"errors\": [\n"
+     "    ],\n"
+     "    \"info\": {\n"
+     "        \"bogus-instance\": {\n"
+     "            \"snapshots\": {\n"
+     "                \"snapshot2\": {\n"
+     "                    \"children\": [\n"
+     "                        \"snapshot3\",\n"
+     "                        \"snapshot4\"\n"
+     "                    ],\n"
+     "                    \"comment\": \"\",\n"
```

Comment:
Hey @sharder996, it looks like the test failures are unrelated to my rebases after all. For instance, `NonOrderableListInfoOutputFormatter/FormatterSuite.properly_formats_output/json_info_single_snapshot_info_reply` fails because the comment is empty here, while it is constructed to have `"This is a comment with some\nnew\r\nlines."` in the test. But let me know if there is something I am overlooking.

Reply:
Hmmm, that's strange... My local branch which does not have the rebase pulled in has the comment correctly populated in [923e355](https://github.com/canonical/multipass/pull/3169/commits/923e35579330a2be936356a0047f6ef1bec8e54b) and all tests are passing.

Reply:
Ohh, should this have been based on a branch other than `snapshots`? The one for snapshot details in the table formatter, perhaps? What does it look like in your local repo?

Reply:
ok, this should be fixed now

Reply:
Great, thank you!